src/Tizen.NUI/src/internal/Interop/Interop.TextUtils.cs

```diff
@@ -126,6 +126,12 @@ internal static partial class RendererParameters
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_RendererParameters_isTextColorSet_get")]
             public static extern bool RendererParameters_isTextColorSet_get(global::System.Runtime.InteropServices.HandleRef jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_RendererParameters_minLineSize_set")]
```

Comment:
Are these native functions merged in csharp-binder? otherwise, this could cause exception when app launching.

Reply:
csharp-binder is merged.